### DISCUSSION TITLE
일라스틱빈스톡으로 올린 인스턴스들을 중지할 수 없나요?

### DISCUSSION THREAD

**Discussion by doingH** [fsoftwareengineer/todo-application#89] [질문과 답변] [answered]
일라스틱빈스톡으로 올린 인스턴스들을 중지할 수 없나요??
콘솔에서 찾아봐도 중지하는법이 없고.. 책에도 실행하는법만있고 중지하는법은 없는것 같아서요

**Comment by fsoftwareengineer** [comment, marked as answer]
일라스틱 빈스톡에서 어플리케이션 전체를 삭제하거나 EC2 > Auto Scaling Group에서 연결된 어플리케이션의 ASG를 찾아 min, max, desired count를 0으로 맞춰주시면 인스턴스가 전부 종료합니다.

**Reply by doingH** [reply]
감사합니다.
